S/SuiteSparse/SPEX/build_tarballs.jl

```diff
@@ -29,11 +28,11 @@ dependencies = append!(dependencies, [
     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"); platforms=filter(!Sys.isbsd, platforms)),
     Dependency(PackageSpec(name="LLVMOpenMP_jll", uuid="1d63c593-3942-5779-bab2-d838dc0a180e"); platforms=filter(Sys.isbsd, platforms)),
     Dependency("GMP_jll"; compat="6.2.1"),
-    Dependency("MPFR_jll"; compat="4.1.1"),
+    Dependency("MPFR_jll"; compat="4.2.1"),
```

Comment:
Does changing the compat version here require also bumping the version of SPEX?

Reply:
Usual reminder that changing these compat bounds (and I'm ***not*** suggesting removing them) will make the package uninstallable in julia v1.11-

Reply:
I'm sorry does that mean only in 1.12 and onwards? Or does it include 1.11? 

I changed the compat bounds again because of FreeBSD. We'd need older MPFR built for freeBSD or to filter on older Julia otherwise

Reply:
> I'm sorry does that mean only in 1.12 and onwards? Or does it include 1.11?

1.11- includes 1.11, so this would be installable only in Julia v1.12+.

> I changed the compat bounds again because of FreeBSD. We'd need older MPFR built for freeBSD or to filter on older Julia otherwise

There's also the option to leave aarch64-freebsd out 🙂 I appreciate building packages for the new platform when it's feasible, but I'm not really sure it's worth wrecking the ecosystem for a platform with 0.5±0.5 users.

Reply:
I can build older versions of things as needed to facilitate making more minimal changes. If you need an older MPFR I can do that, or you can filter out aarch64-unknown-freebsd (but leave x86_64-unknown-freebsd).

Reply:
If you want to build older mpfr that would be great, saves us time going back in the future :)

Reply:
https://github.com/JuliaPackaging/Yggdrasil/pull/9543

Reply:
```suggestion
    Dependency("MPFR_jll"; compat="4.1.1"),
```
Should be all set now (https://github.com/JuliaRegistries/General/pull/116169)